editor/src/components/editor/store/editor-state.ts

```diff
@@ -262,8 +262,8 @@ export interface UserState extends UserConfiguration {
   githubState: GithubState
 }
 
-export interface GithubCommish {
-  name: 'commish'
+export interface GithubCommitAndPush {
+  name: 'commit-and-push'
```

Comment:
🙌 I'm never suggesting a joke name to you again!

Reply:
I am truly sorry 💔